src/components/Authorization/Authorization.module.scss

```
@@ -0,0 +1,149 @@
+@import 'src/styles/color.scss';
+@import 'src/styles/breakpoints.scss';
+
+.container {
+  display: flex;
+  justify-content: center;
+  align-items: end;
+  width: 100%;
+  height: 100%;
+}
+
+@media (min-width: $screen-md-min) {
+  .container {
+    align-items: center;
+    font-size: 2rem;
+  }
+}
```

```suggestion
.container {
  display: flex;
  justify-content: center;
  align-items: end;
  width: 100%;
  height: 100%;
  
  @media (min-width: $screen-md-min) {
      align-items: center;
    font-size: 2rem;
  }
}
```

The same for the rest of the media queries